### PR TITLE
Updated packages with CG vulnerabilities

### PR DESCRIPTION
**Pull request by jadahiya-MSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1. Updated `webpack` package and every package inside of `teamsjs-diagnostics-app`
2. Removed unnecessary `package.lock` files
3. Added `@azure/identity` and `nth-check` to overrides

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -121,7 +121,10 @@
       "underscore": "1.12.1",
       "url-parse": "^1.5.0",
       "word-wrap": "^1.2.4",
-      "y18n": "^4.0.1"
+      "y18n": "^4.0.1",
+      "nth-check@<2.0.1": ">=2.0.1",
+      "@azure/identity@<4.2.1": ">=4.2.1",
+      "fast-xml-parser@<4.4.1": ">=4.4.1"
```
Comment (AE-MS) [inline package.json:127]: Can you add explanation for these to the section below, so we know why they are here and when it would be safe to remove them?

Comment (AE-MS) [inline package.json:127]: It looks like this list was previously alphabetized.

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

File: package.json (hunk 1 above)
Comment (jadahiya-MSFT) [reply package.json:127]: This was auto generated. I'm currently looking into seeing if I can just upgrade a single package that has these as a dependency to forgo needing the overrides.

**Review by AE-MS — APPROVED** [review]
:shipit:

**Review by TrevorJoelHarris — APPROVED** [review]

**Pull request merged**